pkg/cmd/repo/delete/delete.go

```diff
@@ -38,12 +39,14 @@ func NewCmdDelete(f *cmdutil.Factory, runF func(*DeleteOptions) error) *cobra.Co
 	cmd := &cobra.Command{
 		Use:   "delete [<repository>]",
 		Short: "Delete a repository",
-		Long: `Delete a GitHub repository.
-
-With no argument, deletes the current repository. Otherwise, deletes the specified repository.
-
-Deletion requires authorization with the "delete_repo" scope. 
-To authorize, run "gh auth refresh -s delete_repo"`,
+		Long: heredoc.Doc(`
+			Delete a GitHub repository.
+			
+			With no argument, deletes the current repository. Otherwise, deletes the specified repository.
+
+			Deletion requires authorization with the "delete_repo" scope. 
+			To authorize, run "gh auth refresh -s delete_repo"
+		`),
```

Comment:
Minor improvement to bring this in line with formatting done in other usage documentation:

```suggestion
		Long: heredoc.Docf(`
			Delete a GitHub repository.
			
			With no argument, deletes the current repository. Otherwise, deletes the specified repository.

			Deletion requires authorization with the %[1]sdelete_repo%[1]s scope. 
			To authorize, run %[1]sgh auth refresh -s delete_repo%[1]s
		`, "`"),
```